FX/BryanChi_FX Devices.lua

```diff
@@ -55,8 +63,6 @@
 --   Please check the forum post for info:
 --   https://forum.cockos.com/showthread.php?t=263622
 
-
-
 --------------------------==  declare Initial Variables & Functions  ------------------------
 VersionNumber = 'V1.0beta10.3.1 '
```

Comment:
Missing in-script version bump?
```suggestion
VersionNumber = 'V1.0beta10.3.2 '
```

Reply:
Oops, yes,  thank you!